apps/sptribs/sptribs-sptribs-frontend/image-repo.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: image.toolkit.fluxcd.io/v1alpha2
+kind: ImageRepository
+metadata:
+  name: sptribs-sptribs-frontend
+spec:
+  image: hmctspublic.azurecr.io/sptribs/sptribs-frontend
```

Comment:
Actually, this is not right. it doesn't match the image https://github.com/hmcts/sptribs-frontend/blob/master/charts/fis-ds-web/values.yaml#L5

Reply:
I will change it in the corresponding image to match the flux config as its intended to be sptribs-frontend and not fis-ds-web